### ISSUE TITLE
Accessibility - Contrast ratio doesn't meet AA standards

### ISSUE THREAD

**Issue by DimplePatel** [MHRA/products#96]
**Describe the bug**
The colour contrast for the mid blue text colour against the light blue background fails to meet AA accessibility standards. The contrast is 4.32:1 but AA standards are 4.5:1 and above. 
https://webaim.org/resources/contrastchecker/?fcolor=1D70B8&bcolor=E7EBF4 

**To Reproduce**
1) Access https://mhraproductsdev.z33.web.core.windows.net/ for the first time. View the cookies hyperlink on the cookies header
2) Search for a valid phrase and scroll to the list of results

**Expected behavior**
Colour contrast across the site should be 4.5:1 and above to meet AA standards

**Screenshots**
<img width="960" alt="Screenshot 2019-12-10 at 15 40 42" src="https://user-images.githubusercontent.com/6122821/70544163-88db3100-1b63-11ea-950c-a0d37a54548e.png">
<img width="957" alt="Screenshot 2019-12-10 at 15 42 11" src="https://user-images.githubusercontent.com/6122821/70544235-a9a38680-1b63-11ea-91d0-c05872c01f5c.png">

**Additional context**

**Comment by rjkerrison** [comment]
![Screenshot 2019-12-10 at 16 43 44](https://user-images.githubusercontent.com/7150842/70549509-3c482380-1b6c-11ea-88d8-a2d7f0d5b009.png)

Also a problem with Yellow Card.

**Comment by sandracoen1405** [comment]
banner appears 
and only the first time if I click on "accept cookies"
![image](https://user-images.githubusercontent.com/58703117/70554448-0a878a80-1b75-11ea-8545-cc5d7690c4d6.png)

clicking on cookie policy leads to this
![image](https://user-images.githubusercontent.com/58703117/70554488-1bd09700-1b75-11ea-830b-7200eb31461e.png)

**Comment by roughprada** [comment]
Please go back and change few things to we can meet accessibility criteria

All links should be underlined based on the GOV Styles we are following for typography: https://design-system.service.gov.uk/styles/typography/#links

**Cookie banner**
Background colour: remove
Copy colour: #F5F5F5 (MHRA-grey 10%)

That should cover the contrast: https://webaim.org/resources/contrastchecker/?fcolor=1D70B8&bcolor= F5F5F5

Zeplin screens here: https://zpl.io/b6oBlzm

**Search results**
card background to: #F5F5F5 (MHRA-grey 10%)

Zeplin screens here: https://zpl.io/aMnRzPr

**Yellow card**
Change the link for a Primary CTA reading "start now" : https://zpl.io/aX18JQx

**Comment by roughprada** [comment]
Sorry, correction on the cookie banner:

Background colour: #F5F5F5 (MHRA-grey 10%)
Copy colour: #0F1290 (MHRA 100%)

**Comment by DimplePatel** [comment]
cookie link is now at 4.74:1 contrast which meets AA standards. Yellow card link has now been replaced by a button which passes AA standards. Same contrast ratio for search results
